src/components/Project/AragonClient.js

```diff
@@ -253,12 +260,15 @@ class AragonClient extends React.Component {
               <Entry>
                 <Module>
                   <div className="badge-container">
+                    <StyledBadge className="acuamarine">
+                      Identity
+                    </StyledBadge>
                     <AutarkBadge>Autark</AutarkBadge>
                   </div>
-                  <h3>Rich User Profiles</h3>
+                  <h3>Identity: rich user profiles</h3>
                   <h4>
-                    Allow individuals to associate additional details to their
-                    identity such as Github commit history, work history,
+                    Allow individuals to add details to their profile
```

Comment:
> Allow individuals to add details to their profile

I notice that "individuals" was changed to "user" above but not here. Any particular reason why not?